src/Simplex/Chat/Library/Commands.hs

```diff
@@ -3541,8 +3538,11 @@ cleanupManager = do
     cleanupUser cleanupInterval stepDelay user = do
       cleanupTimedItems cleanupInterval user `catchChatError` (toView . CRChatError (Just user))
       liftIO $ threadDelay' stepDelay
+      -- TODO remove in future versions: legacy step - contacts are no longer marked as deleted
```

Comment:
removed?

Reply:
there can still be deleted and not processed contacts in database